webbpsf/tests/test_psfgrid.py

```diff
@@ -204,3 +206,62 @@ def test_saving(tmpdir):
 
     # Remove temporary directory
     tmpdir.remove()
+
+
+def test_2d_to_griddedpsfmodel():
+    """Test that utils.to_griddedpsfmodel function works fof a 2D HDUList input"""
```

Comment:
minor: typo "fof"

Reply:
The motivation for this test is just to ensure there's not any error for the degenerate case of a single-PSF "grid", is that correct?

Reply:
Yes, the creation of a single PSF "grid" is tested in `test_one_psf()` and then this test checks turning the `HDUList` object back into a `GriddedPSFModel` object using `utils.to_griddedpsfmodel` (since it's 1 extra step compared to a 3D input).

Also I'll update that typo real quick